addons/l10n_ua/data/account_tax_template.xml

```diff
@@ -0,0 +1,485 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="ua_psbp_6412" model="account.account.template">
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.acc_tag_vat')])]"/>
+        </record>
+
+        <!-- Tax template for VAT -->
+        <record id="sale_tax_template_vat20_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">10</field>
+            <field name="name">Реалізація ПДВ 20%</field>
+            <field name="description">+ ПДВ 20%</field>
+            <field name="amount">20</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="ua_psbp_6431"/>
+            <field name="refund_account_id" ref="ua_psbp_6431"/>
+            <field name="tag_ids" eval="[(6,0,[ref('l10n_ua.tax_tag_vat20')])]"/>
+            <field name="tax_group_id" ref="l10n_ua.tax_group_vat20"/>
+        </record>
+        <record id="sale_tax_template_vat20incl_psbo" model="account.tax.template">
+            <field name="chart_template_id" ref="l10n_ua_psbo_chart_template"/>
+            <field name="sequence">10</field>
```

Comment:
Some records have the same sequence. I know some localizations need some tax templates that are almost never used, and these should not be displayed first. Do we have this kind of issue with Ukraine? Isn't one of the records preferred to the other?